app/controllers/application_controller.rb

```diff
@@ -4,6 +4,7 @@ class ApplicationController < ActionController::Base
   protect_from_forgery
 
   prepend_before_filter :restrict_iframes
+  before_filter :set_cache_headers # Issue #1213, prevent cart emptying via cache when using back button
```

Comment:
So this will remove all browser caching on all pages?

Reply:
What sort of impact will that have?

Reply:
I'm thinking if this is an issue with Angular not picking up the changes after the back button is used, maybe there's a way to make Angular trigger a cart refresh, and leave the browser caching alone?